domains/authorization/Authorization.API/API.IntegrationTests/API.IntegrationTests/Migrations/AddOrganizationConsentTests.cs

```diff
@@ -39,18 +41,84 @@ public async Task AddTerms_Migration_SetsTermsAcceptedToFalseForExistingOrganiza
         var applyMigration = () => migrator.MigrateAsync("20241021074018_RemoveConsentTable");
         await applyMigration.Should().NotThrowAsync();
 
-        var organizations = await dbContext.Organizations.ToListAsync();
-        var organizationConsents = await dbContext.OrganizationConsents.ToListAsync();
-        var clients = await dbContext.Clients.ToListAsync();
+        var connectionString = dbContext.Database.GetConnectionString();
```

Comment:
This test grew alot. How come?

Reply:
Svært at forklare over tekst. lad os snakke i stedet.